stranger/utils.py

```diff
@@ -135,8 +135,8 @@ def get_repeat_info(variant_info, repeat_info):
         LOG.warning("No info for repeat id %s", repeat_id)
         return None
 
-    rep_lower = repeat_info[repeat_id]['normal_max']
-    rep_upper = repeat_info[repeat_id]['pathologic_min']
+    rep_lower = repeat_info[repeat_id].get('normal_max', 0)
+    rep_upper = repeat_info[repeat_id].get('pathologic_min', 0)
```

Comment:
inf, -1, or some approximation? Will look strange for research vars w/o pathologic_min?